COLLABORATOR_GUIDE.md

```diff
@@ -6,8 +6,7 @@
 * [Accepting Modifications](#accepting-modifications)
  - [Involving the TC](#involving-the-tc)
 * [Landing Pull Requests](#landing-pull-requests)
- - [Technical HOWTO](#technical-howto)
```

Comment:
Sure. I wanted to preserve the information but avoid misguiding the reader into think that the manual procedure should be followed other than if mandated by emergencies.